arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/std/map.kt

```diff
@@ -46,7 +45,7 @@ fun <K, V> PTraversal.Companion.map(): Traversal<Map<K, V>, V> =
 fun <K, V> At.Companion.map(): At<Map<K, V>, K, Option<V>> =
   At { i ->
     PLens(
-      get = { it.getOption(i) },
+      get = { Option.fromNullable(it[i]) },
```

Comment:
I liked more the previous version 😆

Reply:
I'm actually thinking if this should be `At<Map<K, V>, K, V?>` but the `V` should be restricted to `Any`.